memote/support/annotation.py

```diff
@@ -37,6 +37,7 @@
 # 'Kegg'        ['gen','rxn','met'] 'http://www.kegg.jp/'
 # 'SEED'        ['met']             'http://modelseed.org/'
 #
+# 'InChI'       ['met']     'https://www.ebi.ac.uk/chebi/'
```

Comment:
You might wanna contact identifiers.org about that since I didn't decide that:

According to them the main resource for InChiKey's is the NCI:
https://www.ebi.ac.uk/miriam/main/collections/MIR:00000387

While the main resources for InChi are either Chebi or Chemspider:
https://www.ebi.ac.uk/miriam/main/collections/MIR:00000383

Reply:
ChemSpider is a resource for both of them. Since we're only making use of one of the possible resources let's use http://www.chemspider.com/ for both.

Reply:
Since this is only in a comment, not affecting our code at all and dependent on how `identifiers.org` resolves the URIs, I'll respectfully disagree and resolve this comment.